config/controller-service.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Knative Authors LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: tekton-triggers-controller
+  name: tekton-triggers-controller
+  namespace: tekton-pipelines
+spec:
+  ports:
+  - name: metrics
+    port: 9090
+    protocol: TCP
+    targetPort: 9090
```

Comment:
we have been copying this 9090 port thing around (from knative) - do we even need it? I think it might be for emitting metrics that I'm not sure we're actually emitting? (or maybe the knative libs are?)

(just ignore me if you'd rather not look into this)

Reply:
Looks like it's for knative/pkg/metrics, although I'm not sure if we're emitting these or not:
https://github.com/tektoncd/pipeline/blob/master/vendor/github.com/knative/pkg/metrics/config.go#L56

Reply:
nice! thanks for tracking that down :D I've been wondering for so long XD